conformance/tests/gateway-with-attached-routes.yaml

```diff
@@ -86,3 +86,54 @@ spec:
   - backendRefs:
     - name: infra-backend-v1
       port: 8080
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: gateway-with-two-listeners-and-one-attached-route
+  namespace: gateway-conformance-infra
+spec:
+  gatewayClassName: "{GATEWAY_CLASS_NAME}"
+  listeners:
+  - name: http-unattached
+    port: 8080
+    protocol: HTTP
+    allowedRoutes:
+      kinds:
+      - kind: HTTPRoute
+      namespaces:
+        from: Selector
```

Comment:
Since this test is related to the `AttachedRoutes` and not to the `allowedRoutes` I'd lean toward using a more generic approach in relation to the `allowedRoutes` (`from: All` instead of `from: Selector`).

Reply:
Sure mate, done